backend/src/data/repositories/mentees-to-mentors/mentees-to-mentors.repository.ts

```diff
@@ -40,6 +40,24 @@ class MenteesToMentors {
 
     return menteeToMentor ?? null;
   }
+
+  public getMentees(userId: number): Promise<Array<number>> {
+    return this.#MenteesToMentorsModel
+      .query()
+      .select('mentees_to_mentors.mentee_id')
+      .where({ mentorId: userId })
+      .castTo<Array<number>>()
```

Comment:
```suggestion
      .castTo<number[]>()
```

Reply:
Can be a matter of personal preference, but why to type more, if we can use []? :)

Reply:
Just a generic habit from Java :)

Reply:
If coaches ask I change it)

Reply:
Let's make it consistent with other array type definitions

Reply:
should be `number[]`